CivicActions/pgov-cms
# Feature/pgov 313 organizations and divisions

This is a present state / WIP migration of the organizing content types for PGOV. Creates the following migrations, and their associated content types, based on the [U.S. Strategic Portfolio](https://airtable.com/appMitSmbNdNL1EvI/tblkkMOXc5a1jQ3sb/viw4D2VE2vvDQHwKH?blocks=hide) base at Airtable.

```

 -------------- -------- ------- -------------- ------------- --------------------- 
  Migration ID   Status   Total   Imported       Unprocessed   Last Imported        
 -------------- -------- ------- -------------- ------------- --------------------- 
  agencies       Idle     26      26 (100%)      0             2024-11-13 16:57:36  
  coalitions     Idle     4       4 (100%)       0             2024-11-13 16:59:00  
  designations   Idle     2       2 (100%)       0             2024-11-13 16:57:08  
  divisions      Idle     156     157 (100.6%)   -1            2024-11-13 16:58:58  
  goals          Idle     101     113 (111.9%)   -12           2024-11-13 16:59:45  
  milestones     Idle     9       9 (100%)       0             2024-11-13 17:09:36  
  objectives     Idle     292     292 (100%)     -1            2024-11-13 17:09:28  
  plans          Idle     28      28 (100%)      0             2024-11-13 17:09:49  
  projects       Idle     9       9 (100%)       0             2024-11-13 17:09:31  
  subgoals       Idle     682     682 (100%)     0             2024-11-13 17:06:35  
  topics         Idle     34      34 (100%)      0             2024-11-13 16:57:06  
  updates        Idle     85      0              85                                 
 -------------- -------- ------- -------------- ------------- --------------------- 

```

## Testing

1. Ensure you've got a valid Airtable API key in your ddev environment ( See instructions at https://github.com/CivicActions/pgov-cms/pull/12 )
2. Create a fresh install with `drush ci --existing-config`
3. Run the migrations using the helper script from bin/migrate: `ddev migrate`
